### DISCUSSION TITLE
Apple Support for RobVacs in IOS 18.3

### DISCUSSION THREAD

**Discussion by Romeo1984** [Bubba8291/homebridge-sharkiq#35] [Ideas]
@Bubba8291 Are you planning on converting the switches and contacts to native RoboVac widgets when Apple rolls out support for them? https://www.zdnet.com/home-and-office/kitchen-household/ios-18-3-to-bring-home-support-for-robot-vacuums-beta-code-shows/

**Comment by Bubba8291** [comment]
https://github.com/homebridge/HAP-NodeJS/issues/1073
